src/CommonLib/Extensions.cs

```diff
@@ -375,14 +375,26 @@ public static Label GetLabel(this SearchResultEntry entry)
                     objectType = Label.CertTemplate;
                 else if (objectClasses.Contains(PKIEnrollmentServiceClass, StringComparer.InvariantCultureIgnoreCase))
                     objectType = Label.EnterpriseCA;
-                else if (objectClasses.Contains(CertificationAutorityClass, StringComparer.InvariantCultureIgnoreCase))
+                else if (objectClasses.Contains(CertificationAuthorityClass, StringComparer.InvariantCultureIgnoreCase))
                 {
                     if (entry.DistinguishedName.Contains(DirectoryPaths.RootCALocation))
                         objectType = Label.RootCA;
                     else if (entry.DistinguishedName.Contains(DirectoryPaths.AIACALocation))
                         objectType = Label.AIACA;
                     else if (entry.DistinguishedName.Contains(DirectoryPaths.NTAuthStoreLocation))
                         objectType = Label.NTAuthStore;
+                }else if (objectClasses.Contains(OIDContainerClass, StringComparer.InvariantCultureIgnoreCase))
+                {
+                    if (entry.DistinguishedName.StartsWith(DirectoryPaths.OIDContainerLocation,
+                            StringComparison.InvariantCultureIgnoreCase))
+                        objectType = Label.Container;
+                    else
+                    {
+                        if (entry.GetPropertyAsInt(LDAPProperties.Flags, out var flags) && flags == 2)
```

Comment:
flags == 2?

Reply:
Yep, thats the constant value for IssuancePolicy objects according to docs